app/src/main/java/com/example/roome/Apartment_searcher_tabs_classes/EditProfileApartmentSearcher.java

```diff
@@ -170,12 +167,11 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
      */
     private void validateUserFirstName() {
         firstNameEditText = getView().findViewById(R.id.et_enterFirstName);
-        firstNameEditText.setText(asUser.getFirstName() + "***"); //todo
-
+        firstNameEditText.setText(asUser.getFirstName());
```

Comment:
למה לעשות SET TEXT
ואז לבדוק האם הוא תקין?

Reply:
Set text to the user first name.. and check if valid because he can maybe log in with google and the name won't be valid.